css/components/avatar/index.css

```diff
@@ -61,25 +61,21 @@
 }
 
 .avatar-xsm {
-  border-width: 0.15rem;
+  border-width: 2.5px;
 
   width: 3rem;
   height: 3rem;
 }
 .avatar-badge-container {
   position: relative;
-  /* transform:scale(1);
-    border-radius: 50%;
-    transition: all .3s cubic-bezier(0.175, 0.885, 0.32, 1.275);
-    box-shadow: var(--bx-sh-3); */
 }
 .avatar-badge {
   right: 0%;
   top: 0%;
   position: absolute;
   border-radius: 50%;
   min-width: 2rem;
-  padding: 0.25rem;
+  padding: 4px;
```

Comment:
Below 10rem, you have two choices either to use pixels or rems, whatever you choose make it consistent through out, will make the code more uniform and consistent